src/admin-views/aggregator/fields/schedule.php

```diff
@@ -40,7 +40,8 @@ class="tribe-dependent"
 				type="hidden"
 				name="aggregator[<?php echo esc_attr( $origin ) ?>][schedule_day]"
 				id="tribe-ea-field-<?php echo esc_attr( $origin ) ?>_schedule_day"
-				class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-small"
+				class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-medium"
+				style="margin-left: 0;"
```

Comment:
Specifically because this item did not fit any of the other usages of `tribe-ea-field` next to a `tribe-ea-field-readonly`